### ISSUE TITLE
lastgenre: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated

### ISSUE THREAD

**Issue by zhelezov** [beetbox/beets#3194]
### Problem
PyYAML has deprecated the usage of the plain `yaml.load(input)` function, which leeds to a warning from the `lastgenre` plugin:
```
/usr/lib/python3.7/site-packages/beetsplug/lastgenre/__init__.py:146: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  genres_tree = yaml.load(f)
```
Full details in the url from the warning: https://msg.pyyaml.org/load
### Setup

* OS: Fedora 29 
* Python version: 3.7.2
* beets version: 1.4.7
* PyYAML version: 5.1

**Comment by arcresu** [comment]
See #3192 :)

**Comment by sampsyo** [comment]
Indeed! Fixed in #3192.
